test-suite/output/bug_9682.out

```diff
@@ -0,0 +1,13 @@
+File "stdin", line 14, characters 0-349:
+Warning: Lonely notation "mmatch _ in _ as _ return M _ with _ end" was
+already defined with a different format.
```

Comment:
Could we make this warning explain the consequence? (one of the formats is ignored?)

Reply:
Good point. The warning is not introduced in this PR but I may propose to change it for "Overriding previous format of lonely notation "mmatch _ in _ as _ return M _ with _ end". WDYT?

Reply:
Sounds good